meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
 # Explicit machine declaration by build job:
 # MACHINE = "$TARGET_MACHINE"
 
+# Build number shared by all jobs across the different machines:
+# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
+
```

Comment:
External BUILD_ID might be relied on inside ostroproject-ci.inc to set OS_VERSION.
It shouldn't be used anywhere deeper. In future, it would be better that we would generate proper content of OS_VERSION field in dymamic section than making some assumptions on format of string passed externally.